tests/ast/test_ast_driver.py

```diff
@@ -224,6 +226,12 @@ def test_ast(session, tables, test_case):
             ClearTempTables(actual_message)
             ClearTempTables(expected_message)
 
+            # If this is not python 3.9, then for the purposes of the expectation tests we will ignore the line_no
+            # information since it can be different based on various python bug fixes.
+            if sys.version_info.major != 3 or sys.version_info.minor != 11:
```

Comment:
we can assume sys.version_info.major to be 3 (you can add an assert if you feel strongly about safeguarding against 4).

Can we do
`if sys.version_info.minor <= 10:`?

Let's also update the comment.